### ISSUE TITLE
Error when try in raspberry 3

### ISSUE THREAD

**Issue by creativefull** [jaruba/wcjs-player#45]
hi jaruba,. I get an error when I run in raspberry 3.
or maybe I need a WebGL?

**Comment by RSATom** [comment]
it will not work on RPi, at least while X11 will not have hardware accelerated drivers there...

**Comment by jaruba** [comment]
@RSATom although `WebChimera.js VLC` won't work because of the lack of hardware acceleration, I think [WebChimera.js GStreamer](https://github.com/RSATom/wcjs-gs) might, as stated on this [wiki page](https://wiki.matthiasbock.net/index.php/Hardware-accelerated_video_playback_on_the_Raspberry_Pi):

> gstreamer-1.0 features gst-omx, a module which can do hardware-accelerated video decoding on the Raspberry Pi.

@creativefull but the api for `WebChimera.js GStreamer` is too different from the `WebChimera.js VLC`, so `wcjs-player` won't work with it..

**Comment by jaruba** [comment]
@creativefull If you want to test this theory though, you'd need to enable hardware acceleration in `NW.js` too, from what I've read here:

https://github.com/nwjs/nw.js/issues/1151#issuecomment-120854731

It seems that people did manage to get hardware acceleration working in `NW.js` on the RPI before.

**Comment by RSATom** [comment]
I think there are more reasons use QML based players on RPi. I'm working on porting https://github.com/RSATom/WebChimera-desktop to RPi right now.

**Comment by jaruba** [comment]
@RSATom but isn't it still based on VLC, I thought VLC had an issue with hardware acceleration on RPI..

**Comment by RSATom** [comment]
it's only the part of the problem, second part - missing hardware acceleration of frames display on RPi. It could not be fixed only by using GStreamer (which could use hardware decoder) - it require hardware accelerated renderer.

**Comment by creativefull** [comment]
meaning i must be install GStreamer first, ?

**Comment by creativefull** [comment]
yes my RPI did'nt have hardware acceleration ,. just software only ,.

**Comment by RSATom** [comment]
> meaning i must be install GStreamer first, ?

no, we mean you have no any easy way to get it working.

**Comment by creativefull** [comment]
do yo have any solution ?

**Comment by RSATom** [comment]
No, I don't.
And please read this: https://github.com/RSATom/WebChimera.js/wiki/Limitations-%28Read-this-for-Android-|-Raspberry-Pi-|-performance-issues%29

**Comment by creativefull** [comment]
Ok thnks :)
